Workbooks/AKS/Node-Disk-Capacity-AKS-Engine/settings.json

```diff
@@ -1,6 +1,6 @@
 {
     "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
-    "name":"Disk Capacity (Workspace)",
+    "name":"Disk Capacity (AKS-Engine)",
```

Comment:
You don't have to differentiate. A different workbook blade with the corresponding workbooks should be loaded

Reply:
updated